src/iidm/NodeBreakerVoltageLevel.cpp

```diff
@@ -52,6 +55,7 @@ void NodeBreakerVoltageLevel::attach(Terminal& terminal, bool test) {
     if (!test) {
         auto& nodeTerminal = dynamic_cast<NodeTerminal&>(terminal);
         unsigned long node = nodeTerminal.getNode();
+        m_graph.addVertexIfNotPresent(node);
```

Comment:
https://github.com/powsybl/powsybl-iidm4cpp/pull/137#discussion_r451380468
Yes is is here.